lms/static/sass/shared-v2/_layouts.scss

```diff
@@ -2,12 +2,18 @@
 
 .content-wrapper {
     max-width: 1180px;
-    padding-top: 10px;
+    padding-top: ($baseline/2);
+    padding-bottom: ($baseline*2);
```

Comment:
This can also be written:
```padding: {
    top: $baseline/2;
    bottom: $baseline/2;
}
```
Which I think it slightly cleaner, but don't really have a strong opinion on.